sirepo/package_data/static/js/sirepo.js

```diff
@@ -2161,6 +2161,17 @@ SIREPO.app.factory('requestSender', function(cookieService, errorService, userAg
         $window.open(self.formatUrl(routeName, params), '_blank');
     };
 
+    self.openSimulation = (app, route, simId) => {
+        $window.open(
+            self.formatUrl('simulationRedirect', {
+                simulation_type: app,
+                route: route,
```

Comment:
I think this should be local_route/localRoute (everywhere), since that's what it is.

Reply:
> I think this should be local_route/localRoute (everywhere), since that's what it is.

Done almost everywhere;  I excluded e.g. arguments to `reply_redirect_for_local_route` since they have "local _route" in the function name